packages/pigment-css-react/README.md

```diff
@@ -663,6 +663,104 @@ declare module '@pigment-css/react/theme' {
 }
 ```
 
+## Right-to-left support
+
+Pigment CSS is designed for left-to-right (LTR) languages by default. To support right-to-left (RTL) languages, add the `dir="rtl"` attribute to your app's `<html>` element or any other equivalent top level container. Then, update your bundler config as follows:
+
+### Next.js
+
+```js
+const { withPigment } = require('@pigment-css/nextjs-plugin');
+
+// ...
+module.exports = withPigment(nextConfig, {
+  theme: yourCustomTheme,
+  // CSS output option
+  css: {
+    // Specify your default direction
+    defaultDirection: 'rtl',
```

Comment:
I would add the `generateForBothDir: true,` in this first demo and explain it. I hardly suspect anyone would set the `defaultDirection` option without using this option too. The selector option should go in a separate section as it's more or less optional.

Reply:
That's how it was before, but I felt like we were introducing two things simultaneously: configuring the bundler and what this prop is/why it exists. I was missing the hook to say what is the value of it the way it was before, that's why I'm exploring a separate section 🤔

Reply:
Yeah. both options will be used together and `generateForBothDir` in 99% of the case will be `true`.
Also, we already introduced configuring the bundler in the starting section for next and vite. This one focusses more on the `css` option.

Reply:
Gotcha, okay. Take a look again now; I added a title to connect the generated CSS part with the content, so it doesn't look like it is something specifically about the Vite config (unless it is).